### ISSUE TITLE
Booking create - dropping slots

### ISSUE THREAD

**Issue by jchantereau** [jchantereau/tartine#88]
@Eschults When we first select the slot (day + time) and then the number of people, the slot drops and we have to select it again

**Comment by Eschults** [comment]
yep, too dirty to make that work properly, source of bugs in javascript... you have to select the number before the slot, no other way around... perhaps could we hide the calendar until the user selects the number of guests?

**Comment by jchantereau** [comment]
@Eschults It's gonna change the UX too much on that page --> v2
